tracking.js

```diff
@@ -75,21 +77,37 @@ function randNum(){
 
 function clickedT(event){
   randNum();
```

Comment:
You should run `randNum` _after_ incrementing the `clicked` counter. Currently, you have a bug where the user is voting for the item that appears in the next image, not the current image.

That is, if I click on the banana and the next image to appear in the spot is for the baby-sweeper thing, my vote goes to the sweeper instead of to the banana.